ci.nix

```diff
@@ -41,4 +41,16 @@ rec {
     '';
   contract-doc-dev = contract-doc-with-commit null;
   contract-doc-release = { sha, date }@commitInfo: contract-doc-with-commit commitInfo;
+  gh = (pkgs.callPackage "${pkgs.path}/pkgs/applications/version-management/git-and-tools/gh" {
```

Comment:
What's going on here, is there no `gh` in `nixpkgs` that can be easily used like `reuse` and other tools we use?

Reply:
gh is in the nixpkgs, but the version presented in nixpkgs is old and doesn't support releases stuff, so we have to rebuild the newer version ourselves

Reply:
Okay, maybe let's write a comment about that?